### DISCUSSION TITLE
Since version 3.0.0, a custom renderer always needs a componentInitFunction

### DISCUSSION THREAD

**Discussion by mos2st** [dj-fiorex/angular2-smart-table#162] [Q&A] [answered]
Hi, how can I solve this issue? I'm completely lost, also looking at the changes in 3.0.0 (Angular 16 support)
![image](https://github.com/dj-fiorex/angular2-smart-table/assets/139537199/69380fd8-f335-49ad-9fff-6ba625c5a4f5)

**Comment by uap-universe** [comment, marked as answer]
You need to specify a `componentInitFunction`.

You can look up the [Demo Source](https://github.com/dj-fiorex/angular2-smart-table/blob/master/projects/demo/src/app/pages/examples/custom-edit-view/advanced-example-custom-editor.component.ts) for an example.

**Reply by mos2st** [reply]
Thanks for the reply :) It helped me to have a look at the changes, as I already have an `OnComponentInitFunction` and didn't realize the name had changed...
